src/Javascript/Worker.js

```diff
@@ -108,6 +132,17 @@ const main = async (port) => {
 }
 
 
+// Try connecting when browser comes online
+self.addEventListener("online", () => {
+  peers.forEach(peer => {
+    tryConnecting(peer)
```

Comment:
Should this check for already connected peers, in case it's a local peer?

Reply:
That's a slightly complicated situation because we don't track connection `status` globally. One thing we could do is call `keepAlive` when we come online, which would start a ping-reconnect race. But we'd have to temporarily lie about `status`.

How much of an issue is it to connect to an already connected peer? I see we `disconnect` before calling `connect`, so maybe connecting twice has issues?

Reply:
I imagine it would be troublesome when working with a bad internet connection and having a local peer. That said, I honestly don't know what would happen if ipfs is in the middle of a "transferal of data", and you would then do a disconnect.

On the other hand, could this be as simple as filtering out the peer addresses with `localhost` or `0.0.0.0`?

Reply:
I like it, that's a great idea! 👍 

Just pushed a change to do that and added `127.0.0.1` in there as well.